janusgraph-core/src/main/java/org/janusgraph/diskstorage/log/kcvs/KCVSLog.java

```diff
@@ -741,6 +741,9 @@ public void run() {
                 }
                 messageTimeStart = messageTimeEnd;
             } catch (Throwable e) {
+                if (e instanceof PermanentBackendException) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [An instanceof check is being performed on the caught exception.  Create a separate catch clause for this exception type.](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=5026056)